lib/src/components/NxCombobox/NxCombobox.scss

```diff
@@ -60,7 +60,7 @@
   }
 
   .nx-dropdown-button.selected {
-    background-color: var(--nx-swatch-blue-90);
+    background-color: var(--nx-color-interactive-background-selected);
     color: var(--nx-color-text-dark);
```

Comment:
In the mocks the 'selected' text color is `indigo-80`. If the mocks are wrong, then this should be `--nx-color-text-stark`

Reply:
Actually, I suspect the mocks are wrong. All the interaction states show the button to have a text color of `indigo-80` but in NxDropdown's sketch file, hover & click both have the text color set to `indigo-90`

Reply:
I have corrected the mockups. Placeholder text should be indigo-70, and Entry text should be indigo-80. Disabled text should be the same color regardless of what the copy is.

Reply:
> I have corrected the mockups. Placeholder text should be indigo-70, and Entry text should be indigo-80. Disabled text should be the same color regardless of what the copy is.

I assume that is actually in response to the other thread on this PR.  But @JasonDSwearingen, we do need your input on the issue @EmilyBermudez88 described above: what color is the text supposed to be on a selected dropdown menu item (specifically in combobox though we assume it should be the same everywhere)?

Reply:
> In the mocks the 'selected' text color is indigo-80. If the mocks are wrong, then this should be --nx-color-text-stark

For now, changed to use `...-stark`. 5adb2f508

Reply:
> > In the mocks the 'selected' text color is indigo-80. If the mocks are wrong, then this should be --nx-color-text-stark
> 
> For now, changed to use `...-stark`. [5adb2f5](https://github.com/sonatype/sonatype-react-shared-components/commit/5adb2f50842eaf298b06f7a208e7ba6e205c63a7)

Looks good. I just checked the mocks and they now have the text color set to `indigo-90` when selected